src/pages/Home.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from './Home';
+
+it('Home page should render correctly', () => {
+  const tree = renderer.create(<Home />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Nice work adding the Jest Snapshots tests for your React Components. As you also know, one of the major requirements is for you to use React Testing Library to simulate user interactions or verify elements of the components. You can read this article to add simple tests for each component https://www.robinwieruch.de/react-testing-library/